### DISCUSSION TITLE
Nav menu similar to MS Teams

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/fluentui-blazor#2601] [Q&A]
I am trying to design a similar navigation menu to the one in Microsoft Teams. The sidebar should always be visible, even on mobile devices. I also want to include a secondary navigation bar (similar to the Planner sidebar in the screenshot) that is fully collapsible. Additionally, I would like to implement a drop shadow effect between the AppBar, NavBar, and the page content. Can anyone please guide me or provide a sample of how this can be done in Blazor?

Side Menu Visible
![image](https://github.com/user-attachments/assets/aa8ee2e0-720f-42ae-b78a-44255b416808)

Side Menu Collapsed
![image](https://github.com/user-attachments/assets/da23c68f-f982-4cef-b3ab-e76e6b0237d7)

Thanks

**Comment by vnbaaij** [comment]
Have you seen https://www.fluentui-blazor.net/AppBar?

It supports `OnClick` event to hook in whatever code you'd want/like. The shadows can most probably be added with standard CSS rules.

**Reply by ghost** [reply]
Yes I have seen the AppBar component but for the secondary (Fully collapsible menu) is there any component? or NavMenu component support fully collapsible view?

**Reply by vnbaaij** [reply]
Maybe you an use https://preview.fluentui-blazor.net/Splitter#collapsesplitter and then switch panel 1 and 2 with CSS. (not tested that)

**Reply by SpenceDahl** [reply]
Can the orientation of the AppBar be changed? Alternatively, is there a component similar to FluentAppBarItem that can be used in a toolbar?

**Reply by vnbaaij** [reply]
> Can the orientation of the AppBar be changed? 

No, it is designed to be displayed vertically 


>Alternatively, is there a component similar to FluentAppBarItem that can be used in a toolbar?
No, but you can definitely define your own components for that.

**Comment by RobDene** [comment]
NavMenu has the "Hamburger" menu to shrink as standard - would that not work for you ?
